### ISSUE TITLE
Custom Tilesets for Unity Mapbox SDK not working

### ISSUE THREAD

**Issue by caraneel** [mapbox/mapbox-unity-sdk#1547]
Hi, I'm a complete newbie to both Unity and Mapbox but have been able to figure everything out until now. I made two tilesets with custom data, and I followed this tutorial: 
https://docs.mapbox.com/unity/maps/overview/custom-maps/
to try to load my custom data into my Unity project. However, I cannot get the buildings to load in my scene. Of note, I no longer have a "vector" menu under Abstract Maps, this functionality seems to have been shifted to the "Map Layers" menu (I believe I am working with a newer version of the Mapbox SDK).

Instead of using the tutorial tileset ID, I am using mine: caraneel.ck6o5k7x20vaf2npg7s84e6ef-6y2vz and for Property Name I'm using 'graduation_rate.' (I've attached screenshots of all my settings in the Abstract Map)

* Unity version: 2019.2.15f1
* Mapbox SDK version: 2.1.1
* The platform you're building to: iOS but not yet.
* A description of what you're trying to do: create a map of graduation rates by different public schools in NYC
<img width="272" alt="Screen Shot 2020-02-16 at 1 00 18 PM" src="https://user-images.githubusercontent.com/15758690/74609864-4d596780-50bc-11ea-9a96-08dd49936655.png">
<img width="278" alt="Screen Shot 2020-02-16 at 1 00 12 PM" src="https://user-images.githubusercontent.com/15758690/74609866-4df1fe00-50bc-11ea-9d8d-3a88cea45d8c.png">
<img width="285" alt="Screen Shot 2020-02-16 at 12 59 58 PM" src="https://user-images.githubusercontent.com/15758690/74609867-4e8a9480-50bc-11ea-8819-e8007ba94628.png">

**Comment by brnkhy** [comment]
@caraneel one thing caught my eye, `Buildings with unique ids` checkbox shouldn't be checked. It's a little unintuitive sadly, but it only works with data laters support unique ids and custom layers doesn't have that. Can you try that and let me know if it fixes the issue?

**Comment by caraneel** [comment]
hi @brnkhy, I ended up fixing it by translating my data from CSV to geoJSON with the TURF.js library. I think the issue was that my data was point data and needed to be converted to polygon data. Of note, unchecking that box did reveal more of the data that was hidden one I got it to show up!
<img width="1424" alt="Screen Shot 2020-02-18 at 4 53 17 PM" src="https://user-images.githubusercontent.com/15758690/74838676-fd63e600-52f1-11ea-85a2-f4beb9d66247.png">
